src/Uno.UI/UI/Xaml/UIElement.Pointers.Skia.cs

```diff
@@ -0,0 +1,328 @@
+				{
+					foreach(var target in capture.Targets)
+					{
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, target.Element) { CanBubbleNatively = true };
+						TraverseAncestors(target.Element, pointerArgs, e => e.OnNativePointerUp(pointerArgs));
+					}
+				}
+				else
+				{
+					PropagateEvent(args, e =>
+					{
+						// Console.WriteLine($"PointerManager.Released [{e}/{e.GetHashCode():X8}");
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
+						TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerUp(pointerArgs));
+					});
+				}
+			}
+
+			private void CoreWindow_PointerPressed(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerPressed ({args.CurrentPoint.Position})");
+				}
+
+				PropagateEvent(args, e =>
+				{
+					// Console.WriteLine($"PointerManager.Pressed [{e}/{e.GetHashCode():X8}");
+					var pointer = new Pointer(args.CurrentPoint.PointerId, PointerDeviceType.Mouse, false, isInRange: true);
+					var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
+					TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerDown(pointerArgs));
+				});
+			}
+
+			private void CoreWindow_PointerMoved(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerMoved ({args.CurrentPoint.Position})");
+				}
+
+				var pointer = new Pointer(args.CurrentPoint.PointerId, PointerDeviceType.Mouse, false, isInRange: true);
+
+				if (UIElement.PointerCapture.TryGet(pointer, out var capture))
+				{
+					foreach (var target in capture.Targets)
+					{
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, target.Element) { CanBubbleNatively = true };
+
+						TraverseAncestors(target.Element, pointerArgs, e => e.OnNativePointerMove(pointerArgs));
+					}
+				}
+				else
+				{
+					PropagateEvent(args, e =>
+					{
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
+						TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerMove(pointerArgs));
+					});
+				}
+			}
+
+			private void TraverseAncestors(UIElement element, PointerRoutedEventArgs args, Action<UIElement> action)
+			{
+				action(element);
+
+				foreach(var parent in element.GetParents().OfType<UIElement>())
+				{
+					action(parent);
+
+					if (args.Handled)
+					{
+						break;
+					}
+				}
+			}
+
+			private void PropagateEvent(PointerEventArgs args, Action<UIElement> raiseEvent)
+			{
+				if(Window.Current.Content is UIElement root)
+				{
+					PropagageEventRecursive(args, new Point(0, 0), root, raiseEvent);
+				}
+			}
+
+			private bool PropagageEventRecursive(PointerEventArgs args, Point root, UIElement element, Action<UIElement> raiseEvent)
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)